### PR TITLE
Remove/clear user account fix

### PR DESCRIPTION
**Pull request by AyakorK**
Fix Clear User Account previous PR by adapting it to the 0.26

### CODE REVIEW

**Review by paulinebessoles — APPROVED** [review]
Thank you @AyakorK it's all good for the feature!

File: Gemfile [hunk 1]
```diff
@@ -29,7 +29,10 @@ gem "activejob-uniqueness", require: "active_job/uniqueness/sidekiq_patch"
 gem "fog-aws"
 gem "sys-filesystem"
 
+gem "decidim-faceless", git: "https://github.com/digidemlab/decidim-module-faceless", branch: "release/0.26-stable"
 gem "letter_opener_web", "~> 1.3"
+gem "omniauth-oauth2", ">= 1.4.0", "< 2.0"
```
Comment (Quentinchampenois) [inline Gemfile:34]: Does the gem `omniauth-oauth2` and `sprockets` are required for this feature ? 

If it works as expected without, I think we can remove it

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (AyakorK) [reply Gemfile]: After reviewing, gems are deleted !

Thanks for the review !

**Review by eliegaboriau — APPROVED** [review]

**Review by Quentinchampenois — APPROVED** [review]
Thanks !

**Pull request merged**